ui/src/components/Banner.jsx

```diff
@@ -0,0 +1,272 @@
+import React, { useEffect } from 'react';
+
+import logo from './logo.png';
+import './Banner.scss';
+
+export default function Banner() {
+  useEffect(() => {
+    const image = main()
+    return () => { image.remove() }
```

Comment:
not sure what `remove` does, but the image is never added to the DOM so I guess it gets garbage collected once this component is unmounted?